versions.json

```diff
@@ -13,12 +13,6 @@
       "currentMajor": true,
       "previousMinor": true
     },
-    {
-      "version": "8.9.2",
```

Comment:
I'm thinking we'll want to keep until 8.10.0 is available.  This will shut off staging / snapshot builds for 8.9.3 and may cause some issues with downstream dependencies.

Reply:
Good catch. Changed